types/bls-eth-wasm/index.d.ts

```diff
@@ -0,0 +1,111 @@
+declare module "bls-eth-wasm" {
```

Comment:
You should use the types that I'm adding to herumi as a PR.
https://github.com/herumi/bls-eth-wasm/pull/9

Reply:
You haven't added them yet, once they are released we will delete them here